xml/cap_depl_aks.xml

```diff
@@ -904,4 +913,46 @@ awk '($2 ~ /basic/) { system("cf enable-service-access " $1 " -p " $2)}'
    further verify by creating an initial user and posting some test questions.
   </para>
  </sect1>
+ <sect1 xml:id="sec.cap.aks-kubernetes-upgrade">
+  <title>Upgrading An AKS Cluster and Additional Considerations</title>
+
+  <para>
+   When upgrading the &kube; version of your AKS cluster, be sure to enable swap
+   accounting as it is required by &cap;. As part of the initial AKS cluster creation
+   process earlier in this chapter, swap accounting was enabled on all nodes. During the
+   AKS cluster upgrade process, these existing nodes are deleted and replaced by new nodes,
+   running a newer version of &kube;, but without swap accounting enabled. The procedure
+   below describes how to enable swap accounting on the new nodes added during a upgrade.
+  </para>
+  <procedure>
+   <step>
+    <para>
+     Upgrade your AKS cluster by following the process described in
+     <link xlink:href="https://docs.microsoft.com/en-us/azure/aks/upgrade-cluster"/>.
+    </para>
+    <para>
+     Modify the GRUB configuration of each node to enable swap accounting and then
+     reboot all nodes. <literal>RGNAME</literal> and <literal>AKSNAME</literal> will
```

Comment:
```suggestion
     reboot all nodes. <literal>RGNAME</literal> and <literal>AKSNAME</literal> are the values ...
```

Reply:
Resolved in https://github.com/SUSE/doc-cap/pull/460/commits/07ab36972c4ab629b75720fbccca7036bdb09c88